resource-managers/kubernetes/docker/src/main/dockerfiles/spark/bindings/R/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG base_img
+FROM $base_img
+WORKDIR /
+RUN mkdir ${SPARK_HOME}/R
+COPY R ${SPARK_HOME}/R
+
+RUN apk add --no-cache R R-dev
```

Comment:
Should we install a specific version of R here? Perhaps allow it to be parameterized as a build argument with some default.

Reply:
Thoughts @felixcheung ?

Reply:
I think this could make sense to support, the same could be true for Python as well since minor version mismatch can lead to very bad behaviour. It _might_ make sense to do in a follow up PR since we probably can't install all the versions a user might want from apk?

Reply:
Agreed. Minor versions are something we *might* want to support in the future. Let's look to include this in a follow-up PR if there is enough desire in custom version arguments to be passed as build_args to the `bin/docker-image-tool.sh`